src/qt/walletcontroller.cpp

```diff
@@ -190,6 +190,7 @@ WalletControllerActivity::WalletControllerActivity(WalletController* wallet_cont
     , m_wallet_controller(wallet_controller)
     , m_parent_widget(parent_widget)
 {
+    connect(this, &WalletControllerActivity::finished, this, &QObject::deleteLater);
```

Comment:
4517c51b8605fbf91e2319b16d7d5dcf80cb1565

Originally the idea was to allow the user to see activity details and then the user could dismiss or it could auto dismiss. I also think this is an unrelated change - this PR could also connect `LoadWalletsActivity` finished. But this is also fine and can be improved if later the original idea is implemented.